{{cookiecutter.project_slug}}/.golangci.yaml

```diff
@@ -4,6 +4,7 @@ run:
   tests: true
   skip-dirs:
     - .coverage
+  #Any go file with "#+ build integration" in first line won't be linted
```

Comment:
question: Are you suggesting that adding `integration` to this list prevents the linter from inspecting files tagged with `integration` or are you saying that we added `integration` to this list because tagged files wouldn't be inspected otherwise?

Reply:
I meant first one, so I'll reword it to be clearer.